cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4303,6 +4303,14 @@
     </description>
   </property>
 
+  <property>
+    <name>task.worker.container.kill.after.execution</name>
+    <value>true</value>
+    <description>
+      If true, kill the task worker container after it executes a request.
```

Comment:
Totally new idea, not sure if it makes sense in this context but what if we change this to be a count instead of a boolean? 

So the property would be something like `task.worker.container.kill.after.executions` and the value could be `1` if we want to kill after every run. Or the value could be `0` for never killing the pod. This makes it more flexible so we could change it later on to kill after every N requests. Thoughts?

Reply:
IMO, we should restart task workers at least based on time. E.g., every 1 hour.